api/services/FootprintService.js

```diff
@@ -0,0 +1,250 @@
+'use strict';
+
+const Service = require('trails-service');
+const util    = require('util');
+
+/**
+ * @module FootprintService
+ * @description TODO document Service
+ */
+module.exports = class FootprintService extends Service {
+
+  create(modelName, values, options) {
+    let manager = this.app.orm.getManager();
+    let entity  = manager.getEntity(modelName);
+
+    if (!entity) {
+      return this.app.log.error(`No registered entity found for ${modelName}`);
+    }
+
+    let newRecord = this.app.orm.getPopulator(manager).assign(entity, values, null, true);
+
+    manager.persist(newRecord).flush()
+      .then(result => this.app.log.info(`New record created: ${util.inspect(newRecord, false, 8)}`))
+      .catch(this.app.log.error);
+  }
+
+  find(modelName, criteria, options) {
+    let manager = this.app.orm.getManager();
+
+    manager.getRepository(modelName).find(criteria, options)
+      .then(records => {
+        if (!records) {
+          return this.app.log.info('No records found.');
+        }
+
+        this.app.log.info(records);
+      })
+      .catch(this.app.log.error);
```

Comment:
I'm not familiar with trails to this leven, but does this.app.log.error also handle the response to the client properly? Do they get the correct status code + response? If not, we have to make some changes.